src/urllib3/response.py

```diff
@@ -29,6 +30,17 @@
 log = logging.getLogger(__name__)
 
 
+def drain_conn(response):
```

Comment:
(For what it's worth, I now believe having `drain_conn` inside the response class is perfectly fine, I just needed a better name, and we now have that. But keeping the function outside of the class is fine too.)